neural_nets/activations/activations.py

```diff
@@ -128,3 +128,22 @@ def grad(self, x):
 
     def grad2(self, x):
         return np.zeros_like(x)
+
+
+class Elu(ActivationBase):
```

Comment:
```suggestion
class ELU(ActivationBase):
```
Capitalize ELU for consistency with other activations